src/main/java/com/sematext/solr/redis/RetryingCommandHandler.java

```diff
@@ -65,7 +71,17 @@ public RetryingCommandHandler(final JedisPool connectionPool, final int maxRetri
       }
     } while (retries++ < maxRetries);
 
-    // Make the compiler happy
-    return null;
+    if (retries > 0) {
+      log.info("Redis communication error recovered after {}ms and {} of {} retries for command {} on {}",
+          currentTimeMillis() - methodInvocationTime, retries, maxRetries, command.getClass().getName(),
+          localParams.get("key"));
+    }
+
+    return results;
+  }
+
+    /** @return Current time in milliseconds */
+  private static long currentTimeMillis() {
```

Comment:
Isn't this redundant?  Why not call System.currentTimeMillis() directly?

Reply:
Wanted to keep it DRY but I am not married to it 😄